pimd/pim_nb_config.c

```diff
@@ -1882,11 +1882,20 @@ int lib_interface_pim_hello_holdtime_destroy(struct nb_cb_destroy_args *args)
  */
 int lib_interface_pim_bfd_create(struct nb_cb_create_args *args)
 {
+	struct interface *ifp;
+	struct pim_interface *pim_ifp;
+
 	switch (args->event) {
 	case NB_EV_VALIDATE:
 	case NB_EV_PREPARE:
 	case NB_EV_ABORT:
+		/* NOTHING */
+		break;
 	case NB_EV_APPLY:
+		ifp = nb_running_get_entry(args->dnode->parent, NULL, true);
```

Comment:
Could we remove `->parent` from here and all other places? BFD code is the only place in this file that does this dereference.

Reply:
Yes, replaced with `yang_dnode_get_parent`.